oc2ls.md

```diff
@@ -316,35 +316,37 @@ Intellectual Property Rights section of the TC's web page
 
 *This section is normative.*
 
--   **Action**: The task or activity to be performed (e.g., 'deny').
+- **Action**: The task or activity to be performed (e.g., 'deny').
 
--   **Actuator**: The function performed by the Consumer that executes the
-    Command (e.g., 'Stateless Packet Filtering').
+- **Actuator**: The Consumer that executes a Command.
 
--   **Argument**: A property of a Command that provides additional information
+- **Actuator Profile**: The document that defines a category of operations
```

Comment:
Proposed definition:  
**Actuator Profile**: A defined subset of the OpenC2 language (i.e., actions, targets, command arguments, results) plus any extensions required to specify the use of OpenC2 to command a particular function. Actuator Profiles are documented in published specifications and can be represented as schemas.

Reply:
+1 for the definition of ap that @dlemire60 proposed